api.yml

```diff
@@ -17,13 +17,38 @@ tags:
   - name: public
     description: Public APIs facing the Holder and Verifier.
 paths:
-  /credential:
+  /credentials:
     post:
       tags:
         - internal
       summary: Issues a credential and returns it in the response body.
       operationId: issueCredential
       description: Issues a credential and returns it in the response body. Support of this part of the API is REQUIRED for implementations.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/IssueCredentialRequest'
+        description: Parameters for issuing the credential.
+      responses:
+        '201':
+          description: Credential successfully issued!
+          content:
+            application/json:
+              schema:
+                type: object
+                description: The issued credential. This SHOULD include a reference or ID that can be used later for other parts of the API, e.g. changing and retrieving the status of the credential, or refreshing the credential.
```

Comment:
`This SHOULD include a reference or ID`. 

Just to double check, I assume this reference or ID part of the issued credential? (is it correct to assume that the only object returned is a Verifiable Credential.)

Reply:
I don't think this behavior is defined sufficiently... for example, the vc.id property could be this reference, or this reference could be a linked identifier that does not reveal the vc.id...

Reply:
Tracking here: https://github.com/w3c-ccg/vc-issuer-http-api/issues/37

Reply:
Can I resolve this?